docs/make.jl

```diff
@@ -48,6 +48,8 @@ end
 #   "title" => ["subtitle 1" => ("folder 1", "filename 1.jl"),
 #               "subtitle 2" => ("folder 2", "filename 2.jl")]
 files = [
+    # Topic: introduction
+    "Getting started with Trixi" => "getting_started_with_Trixi.jl",
```

Comment:
```suggestion
    "Getting started" => "getting_started.jl",
```
I think it's redundant to refer to Trixi.jl here. Also, I'd rename the file accordingly for consistency